resources/manifests/pod-checkpointer-psp.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: pod-checkpointer-restricted
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default'
+    seccomp.security.alpha.kubernetes.io/defaultProfileName:  'docker/default'
+spec:
+  privileged: false
+  # Required to prevent escalations to root.
+  allowPrivilegeEscalation: false
+  # This is redundant with non-root + disallow privilege escalation,
+  # but we can provide it for defense in depth.
+  requiredDropCapabilities:
+  - KILL
+  - MKNOD
+  - SETUID
+  - SETGID
+  # Allow core volume types.
+  volumes:
+  - 'configMap'
+  - 'hostPath'
+  - 'secret'
+  hostNetwork: true
+  hostIPC: false
+  hostPID: false
+  runAsUser:
+    rule: 'RunAsAny'
+  seLinux:
+    rule: 'RunAsAny'
+  supplementalGroups:
+    rule: 'MustRunAs'
+    ranges:
+    # Forbid adding the root group.
+    - min: 1
+      max: 65535
+  fsGroup:
+    rule: 'MustRunAs'
+    ranges:
+    # Forbid adding the root group.
+    - min: 1
+      max: 65535
+  readOnlyRootFilesystem: false
+  allowedHostPaths:
+  - pathPrefix: "/etc/kubernetes"
+  - pathPrefix: "/var/run"
```

Comment:
Is this used only for this: https://github.com/kubernetes-incubator/bootkube/blob/master/cmd/checkpoint/main.go#L35 ? Can we make it only for that path? Or to a new path that only the pod-checkpointer uses? So it is more contained?

Reply:
Maybe, but then Pod definition needs to be changed as well, which seems to be out of scope of this PR.

Reply:
Oh, don't have the pod definition handy. If it is using this, we may want to keep it like this and open an issue to improve it later :)